src/include/homestore/homestore_decl.hpp

```diff
@@ -89,9 +89,11 @@ ENUM(io_flag, uint8_t,
      DIRECT_IO,   // recommended mode
      READ_ONLY    // Read-only mode for post-mortem checks
 );
-ENUM(blk_allocator_type_t, uint8_t, none, fixed, varsize);
+ENUM(blk_allocator_type_t, uint8_t, none, fixed, varsize, append);
 ENUM(chunk_selector_type_t, uint8_t, // What are the options to select chunk to allocate a block
+     NONE,                           // Caller want nothing to be set
      ROUND_ROBIN,                    // Pick round robin
+     HEAP,                           // Heap chunk selector
```

Comment:
FYI: In next PR, we will remove this HEAP, since Homestore is not aware of how upper layer is building their chunk selector.